en/activity-standard/overview/country-budget-alignment.rst

```diff
@@ -0,0 +1,40 @@
+Country Budget Alignment
+=========
+
+Guidance on implementing budget and aid alignment
+----------
+During the 2.03 upgrade process, decision was made to deprecate BudgetIdentifier codelist. As a result, further guidance was required to provide to publishers on implementing aid and budget alignment.
+This guidance explains how the methodology now works. It stems from work conducted by the IATI TAG 2017 Working Group on Aid and Budget Alignment, and reflects decisions made by the WP-STAT and IATI Steering Committee. In summary:
+
+1) Publish the capital-spend field
+--------------
+For activities at the commitment stage, the percentage of an activity that is capital expenditure should be published using the ``capital-spend`` field. The definition of capital expenditure follows the IMF GFS definition:
+
+  Capital spending is generally defined as physical assets with a useful life of more than one year. But it also includes capital improvements or the rehabilitation of physical assets that enhance or extend the useful life of the asset (as distinct from repair or maintenance, which assures that the asset is functional for its planned life). Capital includes all aspects of design and construction that are required to make the asset operational. Source: D Jacobs, 2009, Capital Expenditures and the Budget, IMF PFM Technical Guidance Note No 8. (IMF, Washington)]
+
+This definition was approved by `WP-STAT under the written procedure <http://www.oecd.org/officialdocuments/publicdisplaydocumentpdf/?cote=DCD/DAC/STAT(2015)30/REV1&docLanguage=En/>`_ in February 2016.
+
+
+2) Publish detailed CRS purpose codes
```

Comment:
There was [a comment on that thread](https://discuss.iatistandard.org/t/guidance-on-implementing-aid-and-budget-alignment/831/17) from @stevieflow that seemed like a good idea -- make more explicit that this should use `sector` rather than `country-budget-items`. So maybe change this line to:

> 2) Publish detailed CRS purpose codes in the `sector` field